src/components/AgreementSections/Lines/Lines.js

```diff
@@ -31,11 +31,14 @@ export default class Lines extends React.Component {
 
   renderAddAgreementLineButton = () => {
```

Comment:
Can we change the name of the function to maybe `renderAddAgreementLineButtonAndBadge` as we have now extended this component